supervisor/supervisor.go

```diff
@@ -274,11 +279,17 @@ func (s *Supervisor) Start() error {
 		"memory":      s.machine.Memory,
 		"cpus":        s.machine.Cpus,
 	}).Debug("containerd: supervisor running")
-	go func() {
-		for i := range s.tasks {
-			s.handleTask(i)
-		}
-	}()
+
+	wg := &sync.WaitGroup{}
+	for i := 0; i < GetDefaultWorkersNum(); i++ {
```

Comment:
same comment as for #852: https://github.com/containerd/containerd/pull/852/files#r116555683

I'm afraid of unforeseen side effects by allowing several tasks to now run in parallel.